### PR TITLE
Technical advisory for Log4J exploit

### PR DESCRIPTION
**Pull request by ianjevans**

### CODE REVIEW

**Review by nickelnickel — APPROVED** [review]
LGTM

File: advisories/aCVE-2021-44228.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+In addition, after careful review of our internal infrastructure we have determined that no deployed configuration of Log4j is exploitable. We will continue to apply regular security patches to all infrastructure and services at our standard cadence.
+
+Customers are strongly encouraged to apply security updates to any Java client applications that use the vulnerable Log4J component as soon as possible. More information about detecting and remediating the vulnerability can be found at the end of this advisory.
+
+## Description
+
+The recently disclosed remote code execution (RCE) vulnerability ([CVE-2021-44228](https://www.cisa.gov/uscert/apache-log4j-vulnerability-guidance)) in Apache’s Log4j software library is currently under exploitation. Log4j is a Java library used by millions of websites, services, and applications to write log messages. A remote unauthenticated actor can exploit this vulnerability by sending a specially crafted request to gain full control of an affected system.
```
Comment (stbof) [inline advisories/aCVE-2021-44228.md:19]: Is the vulnerability really "under exploitation" now? Are attacks currently happening?

Comment (stbof) [reply advisories/aCVE-2021-44228.md:19]: I guess it is.

**Review by stbof — APPROVED** [review]

File: advisories/aCVE-2021-44228.md (hunk 1 above)
Comment (ianjevans) [reply advisories/aCVE-2021-44228.md:19]: Yes, it's pretty bad. https://threatpost.com/apache-log4j-log4shell-mutations/176962/

**Pull request merged**